modules/programs/neovim.nix

```diff
@@ -58,6 +58,28 @@ in
           List here Python 3 packages required for your plugins to work.
         '';
       };
+
+      configure = mkOption {
+        type = types.nullOr types.attrs;
+        default = null;
+        example = literalExample ''
+          configure = {
+              customRC = $''''
+              " here your custom configuration goes!
+              $'''';
+              packages.myVimPackage = with pkgs.vimPlugins; {
+                # loaded on launch
+                start = [ fugitive ];
+                # manually loadable by calling `:packadd $plugin-name`
+                opt = [ ];
+              };
+            };
+        '';
+        description = ''
+          Generate your init file from your list of plugins and custom commands, 
+          ands loads it from the store via `nvim -u /nix/store/hash-vimrc`
```

Comment:
`ands` → `and`

``nvim -u /nix/store/hash-vimrc``→ `<command>nvim -u /nix/store/hash-vimrc</command>`